decidim-core/lib/decidim/resourceable.rb

```diff
@@ -63,12 +63,21 @@ def link_resources(resources, link_name, data = {})
         transaction do
           resource_links_from.where(name: link_name).delete_all
           Array.wrap(resources).each do |resource|
-            Decidim::ResourceLink.create!(
-              from: self,
-              to: resource,
-              name: link_name,
-              data: data
-            )
+            payload = {
+              from_type: self.class.name,
+              from_id: id,
+              to_type: resource.class.name,
+              to_id: resource.id
+            }
+            event_name = "decidim.resourceable.#{link_name}.created"
+            ActiveSupport::Notifications.instrument event_name, this: payload do
```

Comment:
This looks like it's not being tested

Reply:
It is tested in `resourceable_spec.rb:20` at `context "when linking to a resource"`.

Reply:
👍